### DISCUSSION TITLE
Windows 7 support [May 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5731] [Discord Threads]
> 🙋‍♂️ **kiyoko shimizu**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### <@746534006945873982> We haven't tried. Do let us know if you try it out  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1097352734597972008)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
